website/content/docs/plugins/plugin-development.mdx

```diff
@@ -75,6 +75,22 @@ func main() {
 And that's basically it! You would just need to change `myPlugin` to your actual
 plugin.
 
+### Implementing the Version interface
```

Comment:
nit: I would either put this below "Plugin Backwards Compatibility with Vault" or above "Serving A Plugin with Multiplexing" since those two sections are related.